### ISSUE TITLE
about task router

### ISSUE THREAD

**Issue by woshifujiale** [MergeVLA/MergeVLA#4]
你好，我觉得这个融合VLA的研究很有意思，但是我有几个问题想请教一下，就是你们是如何设计task router的，路由的成功率你们测出来的有多少呢？我自己复现下来看好像不理想，就是在LIBERO上路由的，同一个任务不同episode都会出现不同结果。就是我想请教一下目前有对这个方面的改进计划吗？

**Comment by woshifujiale** [comment]
可能是我代码复现的问题，希望作者大大解答一下

**Comment by Daisy-zzz** [comment]
Hi,
In practice, we only run the router on the first episode of each task, so applying it to every episode may introduce some instability. We have also observed that the results can sometimes vary across different machines, since two similar tasks might have almost the same logits.
Since the current router is training-free, its performance is quite sensitive to the dataset. We understand this is a limitation, and we are currently exploring some post-training methods to improve its stability.
